client/internal/http/client.go

```diff
@@ -63,6 +63,25 @@ func New(
 	}, nil
 }
 
+func MustNew(
```

Comment:
what is the purpose of `MustNew`? I believe there could be a benefit in enhancing the naming clarity within this file

Reply:
`MustXXX` is a pattern commonly used in golang that means: return the result only, and call `panic` if any error.

Reply:
oh good to know